examples/sops/local-configs/example-use.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: my-config
+  annotations:
+    config.k8s.io/function: |
+      container:
+        image: gcr.io/kpt-functions/sops
+    config.kubernetes.io/local-config: "true"
+data:
+---
+myapp1: ENC[AES256_GCM,data:zlGNmhTYX5xol4ZZFsiaoGkD73nn,iv:ql9mkhoU1I64E/FJi3iA0HaAe2U3kQVFee2ZLwPnBik=,tag:SqVSfu/JkRrwqidAT/i0pg==,type:str]
```

Comment:
Any reason this is not a K8S resources, i.e. doesn't have GVKN?

Reply:
just to demonstrate that it's possible to decrypt any documents.
I can add both GVKN and non-GVKN,
or change it to GVKN-only.
Can do it along with splitting the multi-doc

Reply:
The spec requires that items in ResourceList be K8S resources:
https://github.com/kubernetes-sigs/kustomize/blob/master/cmd/config/docs/api-conventions/functions-spec.md

Currently, non-K8S resources not leading to validation failures is an implementation detail that shouldn't be depended on. So, please use GVKN only

Reply:
agree, even though it's working now, it's better to follow the specs so it would continue work in future :). I'll remove non-GVKN item.

Reply:
done